core/src/main/java/com/scalar/db/storage/jdbc/JdbcAdmin.java

```diff
@@ -645,7 +645,7 @@ public void close() {
     try {
       dataSource.close();
     } catch (SQLException e) {
-      logger.error("Failed to close the dataSource", e);
+      logger.warn("Failed to close the dataSource", e);
```

Comment:
Strictly speaking, this isn't related to this PR, but just for consistency with other error handlings.